### ISSUE TITLE
jspm -v causing Unexpected token { SyntaxError for 2.0.0-beta.1 but works with 0.16.54

### ISSUE THREAD

**Issue by Ogglas** [jspm/jspm-cli#2453]
`jspm -v` causing `Unexpected token {` SyntaxError for `2.0.0-beta.1 `but works with `0.16.54`

[package.zip](https://github.com/jspm/jspm-cli/files/3126359/package.zip) contains `config.js` and `package.json` samples. 

```
PS C:\Users\oscar\Documents\Test> npm install -g jspm@0.16.54
C:\Users\oscar\AppData\Roaming\npm\jspm -> C:\Users\oscar\AppData\Roaming\npm\node_modules\jspm\jspm.js
+ jspm@0.16.54
added 87 packages, removed 129 packages and updated 20 packages in 20.076s
PS C:\Users\oscar\Documents\Test> jspm -v
0.16.54
Running against global jspm install.
PS C:\Users\oscar\Documents\Test> npm install -g jspm
C:\Users\oscar\AppData\Roaming\npm\jspm -> C:\Users\oscar\AppData\Roaming\npm\node_modules\jspm\bin\jspm
+ jspm@2.0.0-beta.1
added 129 packages, removed 87 packages and updated 20 packages in 15.327s
PS C:\Users\oscar\Documents\Test> jspm -v
C:\Users\oscar\AppData\Roaming\npm\node_modules\jspm\lib\cli.js:647
                catch { }
                      ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Liftoff.jspmCLI.launch.env (C:\Users\oscar\AppData\Roaming\npm\node_modules\jspm\bin\jspm:45:5)
PS C:\Users\oscar\Documents\Test>

```

**Comment by guybedford** [comment]
Thanks, I've added a fix for this in https://github.com/jspm/jspm-cli/commit/e06ef4f8063ed54baf1d3095208c84748ea25abf.

If you run Node.js 11 or 12 you should also not get this error since it is a new language feature not to need the catch binding.

**Comment by Ogglas** [comment]
@guybedford Thanks!

**Comment by guybedford** [comment]
Released in 2.0.0-beta.1.

**Comment by guybedford** [comment]
(although note that `jspm v` is the usage in jspm 2.0 over `jspm -v`)
